corehq/apps/app_manager/templates/app_manager/v1/partials/case_list_properties.html

```diff
@@ -66,23 +66,29 @@
                         <td class="col-sm-3" data-bind="jqueryElement: header.ui"></td>
                         {{ request|toggle_tag_info:"DISPLAY_CONDITION_ON_NODESET" }}
                         {% if request|toggle_enabled:'DISPLAY_CONDITION_ON_NODESET' %}
+                        <td class="col-sm-3 form-group">
+                             <div data-bind="jqueryElement: relevant.ui"></div>
+                         </td>
                         <td class="col-sm-3 form-group" data-bind="css: {'has-error': showWarning}">
                         {% else %}
                         <td class="col-sm-5 form-group" colspan="2" data-bind="css: {'has-error': showWarning}">
                         {% endif %}
                             <div data-bind="jqueryElement: nodeset.ui"></div>
-                        </td>
-                        {{ request|toggle_tag_info:"DISPLAY_CONDITION_ON_NODESET" }}
-                        {% if request|toggle_enabled:'DISPLAY_CONDITION_ON_NODESET' %}
-                        <td class="col-sm-3 form-group">
-                            <div data-bind="jqueryElement: relevant.ui"></div>
                             <span class="help-block"
-                                  data-bind="text: 'A display condition can be specified.'"></span>
+                                  data-bind="text: 'A nodeset must be specified.', visible: showWarning"></span>
                         </td>
-                        {% endif %}
                     <!--/ko-->
                     <!--ko ifnot: $data.hasNodeset -->
+                        {{ request|toggle_tag_info:"DISPLAY_CONDITION_ON_NODESET" }}
```

Comment:
@orangejenny there is a difference in their css.  One way to combine them to one would be to put if conditions for the styling. I thought i was better this way.